src/frontend/components/SøknadsSteg/OmDeg/useOmdeg.tsx

```diff
@@ -98,11 +105,54 @@ export const useOmdeg = (): {
         borPåRegistrertAdresse.verdi === ESvar.NEI
     );
 
-    const komTilNorgeDato = useDatovelgerFeltMedJaNeiAvhengighet(
+    const komTilNorgeDatoVetIkke = useFelt<ESvar>({
+        verdi: søker.komTilNorgeDato.svar === AlternativtSvarForInput.UKJENT ? ESvar.JA : ESvar.NEI,
+        feltId: OmDegSpørsmålId.komTilNorgeDatoVetIkke,
+        skalFeltetVises: avhengigheter => {
+            return (
+                avhengigheter &&
+                avhengigheter.værtINorgeITolvMåneder &&
+                avhengigheter.værtINorgeITolvMåneder.verdi === ESvar.NEI
+            );
+        },
+        avhengigheter: { værtINorgeITolvMåneder },
+    });
+
+    const komTilNorgeDato = useDatovelgerFeltMedUkjent(
         søker.komTilNorgeDato,
-        ESvar.NEI,
-        værtINorgeITolvMåneder,
+        søker.komTilNorgeDato.svar === AlternativtSvarForInput.UKJENT
+            ? ''
+            : søker.komTilNorgeDato.svar,
+        komTilNorgeDatoVetIkke,
         'omdeg.opphold-sammenhengende.dato.feilmelding',
+        værtINorgeITolvMåneder.verdi === ESvar.NEI,
+        false,
+        dagensDato()
```

Comment:
Ville dobbeltsjekket med fag hva som skal være begrensning på disse to feltene :) Nå kan man i teorien si at man komTilNorge og reisteFraNorge samme dag 🤔